README.md

```diff
@@ -28,29 +28,62 @@ application, each tutorial is constructed as hybrid theory and programming in an
 interactive environment, removing the gap between theory and implementation.
 
 If you have comments, questions, or would like to contribute to the project
-please feel free to email [me](mailto:dgasmith@gatech.edu).
+please see our [contributor guidelines]().
 
 ### Getting Started
 
-New users can either follow the
-[Getting Started](https://github.com/psi4/psi4numpy/blob/master/Tutorials/01_Psi4NumPy-Basics/1a_Getting-Started.ipynb)
-notebook or follow the less verbose directions below.
+1. Obtain required software:
+- [Psi4](http://psicode.org/psi4manual/1.1/build_obtaining.html)
+- [Python](https://python.org) 2.7+
+- [NumPy](http://www.numpy.org) 1.7.2+
+- [Scipy](https://scipy.org) 0.13.0+
+ 
+2. Update Psi4 to the development version with Conda:
+```
+conda update psi4 -c psi4/label/dev
+```
 
-To begin using Psi4NumPy:
-- Obtain recent versions of Psi4, NumPy, and SciPy (see Requirements for version details)
-- Locate the `Psi4_install/lib` directory and execute the following: ```export PYTHONPATH=Psi4_install/lib```
+3. Link PsiAPI
+    1. Find appropriate paths
```

Comment:
Yeah, I may have written the build docs there before I realized that reference implementations weren't just a staging ground for tutorials. I have trouble finding that page myself.